### ISSUE TITLE
CSS for programs

### ISSUE THREAD

**Issue by rbeezer** [rbeezer/mathbook#831]
More from #732 

PTX has XSLT for `pre.prettyprint` and `pre.plainprint`:
```xslt
<pre class="{$classes}" style="font-size:80%">
```
Appears current CSS has a `110%` font-size.  Can we coordinate and remove the PTX `style` and move the CSS to something below `100%`?  Width is at a premium and nobody has complained about the smaller font-size here.  I'm pretty sure it is similar in LaTeX in an attempt to keep line breaks and margins, etc somewhat consistent.

Status quo:
http://mathbook.pugetsound.edu/examples/sample-article/html/section-programs.html

**Comment by davidfarmer** [comment]
In Bob's book the inline style has 110% on the code.  See
listing 10.3.1 here:

http://bob.cs.sonoma.edu/IntroCompOrg-RPi/sec-stack-manage.html

Probably that (or else my eye) told me that 110% was good.

I just checked, and it seems like 80 characters fit in the
allocated width if the font size is 90%.  Note that 79 characters
is the official maximum for python style.

So, I set the CSS to 92% and the inline font-size will be
removed?  (for both .prettyprint and .plainprint)

It is good if Bob, who is writing in a language that
tends to have shorter lines, has the option to keep giving 110%.

I just noticed that on a narrow screen the code was cut off,
so I just set it to overflow:auto, and added the gradient
indicator for when smart phones don't see the scroll bar.
